.travis.yml

```diff
@@ -17,15 +17,7 @@ install:
   - pip install codecov
 
 script:
- - python unitTests.py
- - python simple_unit_tests.py
- - PYTHONPATH=. python examples/numerics.py
- - PYTHONPATH=. python examples/TAP.py
- - PYTHONPATH=. python examples/romanNumerals.py
- - PYTHONPATH=. python examples/sexpParser.py
- - PYTHONPATH=. python examples/oc.py
- - PYTHONPATH=. python examples/delta_time.py
- - PYTHONPATH=. python examples/eval_arith.py
+ - python -m unittest discover tests
```

Comment:
Simplified running of tests.  There is a new test class that will run examples as test for us.  I assume an example will fail if it can not parse.